content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -29,6 +29,21 @@ Kubernetes, on Linux:
 - [Docker Engine](#docker)
 - [Mirantis Container Runtime](#mcr)
 
+{{< note >}}
+Docker container runtime (Dockershim) has been [deprecated](
+https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
+as of [version 1.20](
+https://kubernetes.io/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
```

Comment:
We use relative paths rather than absolute ones. Also please help make sure the link appears on one line.
```suggestion
/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
```